ordering/__init__.py

```diff
@@ -67,6 +67,15 @@ def __contains__(self, item: _T) -> bool:
 
         return item in self._labels
 
+    def __eq__(self, other: object) -> bool:
+        if self is other:
+            return True
+        if not isinstance(other, Ordering):
+            return NotImplemented
```

Comment:
Yes I'd say so, that's the behaviour you get when subclassing built-in types. Additionally, if the subclass has altered enough such that the two shouldn't be considered equal, then it should be up to the subclass to define it's own `__eq__` method.